server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -427,7 +428,7 @@ private FieldType resolveFieldType(
             final IndexMode indexMode,
             final String fullFieldName
         ) {
-            if (FieldMapper.DOC_VALUES_SPARSE_INDEX.isEnabled()
+            if (IndexSettings.DOC_VALUES_SKIPPER.isEnabled()
```

Comment:
This change will not be here once we merge https://github.com/elastic/elasticsearch/pull/121751